### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by Epictyphlosion** [vampirefrog/mdxtools#13]
The programs have been updated a lot since the last alpha, and I'm unable to compile any since my main computer isn't set up for compiling programs at the moment. Plus, I would love to test out mdx2vgm.

**Comment by vampirefrog** [comment]
yep, as I expected, it doesn't currently compile, so a new release will have to wait until I get more time

**Comment by vampirefrog** [comment]
technically I could just add vgm logging to mdxplay and that might be good enough

**Comment by vampirefrog** [comment]
Added issue #15
